### DISCUSSION TITLE
Integration with React Native / Expo

### DISCUSSION THREAD

**Discussion by hosenur** [appwrite/appwrite#5117] [Ideas]
An official integration git to use Appwrite with React Native / Expo

**Comment by stnguyen90** [comment]
@HOSENUR, thanks for opening this! 🙏🏼 What kind of integration are you looking for exactly?

**Reply by hosenur** [reply]
User Authentication, all the other can be accomplished by the JS SDK

**Reply by alfirus** [reply]
@HOSENUR , you mean https://github.com/appwrite/sdk-for-web?

**Reply by hosenur** [reply]
This [issue](https://github.com/appwrite/appwrite/issues/2993) has been raised many times , **OAuth** can not be implemented with the [Web SDK]( https://github.com/appwrite/sdk-for-web), rest all will work fine.
Along with Flutter, React Native / Expo is also largely used  to build mobile apps, supporting those will be really helpful

**Comment by ChronoByteCosmonaut** [comment]
That would be awesome 🤞
